### PR TITLE
Add separate internal development kustomize context

### PR DESCRIPTION
**Pull request by bcmendoza**

### CODE REVIEW

File: config/context/kubernetes-dev/patches.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: Deployment
+metadata:
+  name: gm-operator
+  namespace: gm-operator
+spec:
+  template:
+    spec:
+      containers:
+      - name: operator
+        image: "docker.greymatter.io/internal/gm-operator:latest"
```
Comment (dontlaugh) [inline config/context/kubernetes-dev/patches.yaml:11]: does this need to change, too

Comment (dontlaugh) [reply config/context/kubernetes-dev/patches.yaml:11]: or this is the one you're _adding_?

**Review by dontlaugh — COMMENTED** [review]
seems fine :+1:

File: config/context/kubernetes-dev/patches.yaml (hunk 1 above)
Comment (bcmendoza) [reply config/context/kubernetes-dev/patches.yaml:11]: This is the one I'm adding, and it's for internal development use only.

**Comment by bcmendoza** [comment]
@dontlaugh Those who want a "quick install" environment can use the `kubernetes` context. `kubernetes-dev` context will be used by the dev scripts.

**Review by danielpcox — APPROVED** [review]
Yessssss... works for me. Thanks Brian.

**Pull request merged**